packages/core/src/storage.ts

```diff
@@ -112,6 +124,10 @@ export class MemoryStorage implements Storage {
   key(index: number): string | null {
     return Array.from(this.data.keys())[index];
   }
+
+  keys(): string[] {
+    return Array.from(this.data.keys());
+  }
```

Comment:
need to add this to [`SyncSecureStorage`](https://github.com/medplum/medplum/blob/e283645d5444c344844d897b29120455da1ba2f1/packages/expo-polyfills/src/index.ts#L172) as well